src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="MSBuild.Sdk.Extras/2.1.2">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0-ios;net6.0-android;net6.0-maccatalyst</TargetFrameworks>
+    <TargetFrameworks>net6.0-ios14.0;net6.0-android30.0;net6.0-maccatalyst13.5</TargetFrameworks>
```

Comment:
We need these to adhere with .NET MAUI:

```xml
<TargetFrameworks>net6.0-ios10.0;net6.0-android21.0;net6.0-maccatalyst10.13</TargetFrameworks>
```

Reply:
See this conversation for more details: https://github.com/CommunityToolkit/Maui/pull/1#discussion_r688727400

Reply:
For iOS and Android that might work, for maccatalyst it does not: https://dev.azure.com/dotnet/CommunityToolkit/_build/results?buildId=56105&view=logs&j=c44e3ce6-80b8-5090-9435-9627042472dc&t=01dd27be-fb26-56b1-ffa1-33467f7cc411

Reply:
I think the catalyst version number != mac OS version number, to make it even more confusing :D

Reply:
Ah - maccatalyst version matches the iOS version, not the macOS version

https://stackoverflow.com/a/63598566/5953643

Reply:
👇 Let's try this instead
```xml
<TargetFrameworks>net6.0-ios10.0;net6.0-android21.0;net6.0-maccatalyst10.0</TargetFrameworks>
```

Reply:
Wait - what? This says that `net6.0-android30.0` is the _only valid version_?

<img width="492" alt="image" src="https://user-images.githubusercontent.com/13558917/129410019-211cdc58-3126-4c49-b538-f8efbe7f674b.png">

Reply:
Update from the Engineering team: "Don't use version numbers"